python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -323,12 +323,23 @@ def placement_group_deserializer(pg_tuple):
         )
         nodes_summary = freq_of_dicts(self.static_resources_by_ip.values())
 
+        usage_by_node = {}
+        for ip, totals in self.static_resources_by_ip.items():
+            available = self.dynamic_resources_by_ip.get(ip, {})
+            usage_by_node[ip] = {}
+            for resource, total in totals.items():
+                usage_by_node[ip][resource] = (
+                    total - available.get(resource, 0),
+                    total,
+                )
+
         return LoadMetricsSummary(
             usage=usage_dict,
             resource_demand=summarized_demand_vector,
             pg_demand=summarized_placement_groups,
             request_demand=summarized_resource_requests,
             node_types=nodes_summary,
+            usage_by_node=usage_by_node,
```

Comment:
should we remove usage_dict? (and aggregate this before returning?)

Reply:
hmm would need to double check on kuberay's version guarantee story before doing that.

Reply:
Recommended and default usage as of Ray 2.0.0 is to match Ray and autoscaler images.

However, better to be careful and leave it as is for now, anticipating potential for version mismatch.

Reply:
I do need to feature flag filling this field out i think because it might be big data on a big cluster